pandas/core/indexes/base.py

```diff
@@ -2005,15 +2005,37 @@ def __contains__(self, key):
             return False
 
     _index_shared_docs['contains'] = """
-        return a boolean if this key is IN the index
+        Return a boolean if this key is in the index.
 
         Parameters
         ----------
-        key : object
+        key : label
+            The key requested. Immutable-like, 1-dimensional.
 
         Returns
         -------
-        boolean
+        bool
+            Result of the key search.
+
+        See Also
+        --------
+        CategoricalIndex : Returns a bool if the 1-dimensional, Categorical key
```

Comment:
`CategoricalIndex` is the class, do you mean `CategoricalIndex.contains` here?

Reply:
Yes, I mean this. Apparently, there isn't any docstring written for the same. Should I write something like 
`CategoricalIndex : Its :method: contains returns a bool if the key is in index. The key can be of the type same as this class, like the Index.contains` 
or should I still refer to `CategoricalIndex.contains : Returns a bool if the key is in index . . .`

Reply:
The `Index` class is a parent of `CategoricalIndex` as well as all the other index classes like `Float64Index`... It could make sense to list `CategoricalIndex.contains`, `Float64Index.contains`... but I don't think it adds that much value in this case, and it adds noise to the docstring.

I think the best is just leave `Index.isin`, but it's up to you.

Reply:
I agree as `CategoricalIndex.contains` behaves similarly to `Index.contains`. Let's just keep `Index.isin`.